accounting/filters/OsgScheddCpuFilter.py

```diff
@@ -835,19 +836,27 @@ def compute_custom_columns(self, data, agg, agg_name):
                 row["Output MB / File"] = output_mb / output_files
                 row["Total Files Xferd"] = row.get("Total Files Xferd", 0) + output_files
 
-        row["OSDF Files Xferd"] = osdf_files_count or ""
-        if osdf_files_count > 0 and osdf_bytes_total > 0 and total_files > 0 and total_bytes > 0:
-            row["% OSDF Files"] = 100 * osdf_files_count / total_files
-            row["% OSDF Bytes"] = 100 * osdf_bytes_total / total_bytes
+        if osdf_files_count == 0 or osdf_bytes_total == 0:
+            version_string = str(data["CondorVersion"])
+            version_numbers = version_string.split()[0].split(".")
```

Comment:
The second token contains the version number. Let's turn it into a tuple.
```suggestion
            condor_version = tuple([int(x) for x in data["CondorVersion"].split()[1].split(".")])
```